packages/frontend-2/components/project/List.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <section>
+    <div class="flex items-center justify-between gap-4 my-8">
+      <div class="flex items-center gap-4">
+        <h1 class="h4 text-foreground font-bold flex items-center">
+          Projects
+          <span class="ml-2 caption text-foreground-dim">{{ count }}</span>
+        </h1>
+        <button
+          class="rounded-lg bg-blue-100 hover:bg-blue-200 dark:bg-slate-800 hover:dark:bg-slate-900 text-blue-500 xxxdark:text-gray-500 text-sm shadow hover:shadow-md px-4 py-2 cursor-pointer transition"
```

Comment:
1. multiple non-semantic colors here, does it really make sense to have a completely unique button just for this? why not use one of the button types we already have, or introduce a new variation to it that could then be used elsewhere as well?
2. what's the "xxxdark" thing mean? 🤔

Reply:
needs to be migrated to an actual button